protographic/tests/operations/enum-support.test.ts

```diff
@@ -0,0 +1,715 @@
+import { describe, expect, test } from 'vitest';
+import { compileOperationsToProto } from '../../src/operation-to-proto';
+import { expectValidProto, loadProtoFromText } from '../util';
+
+describe('Enum Support', () => {
+  describe('Enums in Query Variables', () => {
+    test('should handle enum variable in query', () => {
+      const schema = `
+        type Query {
+          users(status: UserStatus): [User]
+        }
+        
+        enum UserStatus {
+          ACTIVE
+          INACTIVE
+          PENDING
+        }
+        
+        type User {
+          id: ID!
+          name: String
+          status: UserStatus
+        }
+      `;
+
+      const operation = `
+        query GetUsers($status: UserStatus) {
+          users(status: $status) {
+            id
+            name
+            status
+          }
+        }
+      `;
+
+      const { proto, root } = compileOperationsToProto(operation, schema);
+
+      expectValidProto(proto);
+
+      // Should contain enum definition
+      expect(proto).toContain('enum UserStatus {');
```

Comment:
Using inline snapshots here would be easier read, especially if indentation is also correct.